### DISCUSSION TITLE
Evo Sublime implementation?

### DISCUSSION THREAD

**Discussion by HaikenEdge** [chummer5a/chummer5a#4818] [General]
The Evo Sublime cyberdeck doesn't seem to appear in Chummer5; it's printed in Killing Code, on page 62, and has the distinction of being only available as an implanted cyberdeck, but it's not available under either Gear or Cyberware.

I'm using the latest current non-nightly version.

**Comment by chummer5a** [comment]
RAW interpretation is that it uses standard implanted deck rules; take the cyberdeck object, add evo sublime as a child.
